### DISCUSSION TITLE
Not compatible with recurring tasks?

### DISCUSSION THREAD

**Discussion by sdquinn** [stancl/things3-trello-sync#5] [Support]
Hi there -- so grateful this exists.

Just tried to run an initial force sync after my first task synced over, and got the following message on a tasks I've set to recur in Things -- is this not compatible with recurring tasks in Things?

`@Seans-MacBook-Pro-2` things3-trello-sync-master % php artisan things:sync --force
Syncing all (5515) tasks
Checked task SawEEte2RRJjnkhKuy1SJk (Start planning your honeymoon)
Skipped task SCyvhicDFeqMTfnC1ZrR3n (Go for an intentional walk of at least 5 minutes for 7 days in a row.)
 ErrorException 
  Attempt to read property "parentProject" on string
  at app/Models/Task.php:94
     90▕         $default = config("trello.{$category}.default");
     91▕ 
     92▕         $title = match (config("trello.{$category}.things")) {
     93▕             'heading' => $this->heading?->title,
  ➜  94▕             'project' => $this->parentProject?->title ?? $this->heading?->parentProject->title,
     95▕             'area' => $this->parentArea?->title ?? $this->parentProject?->parentArea->title,
     96▕             default => null,
     97▕         } ?? $default;
     98▕ 
  1   app/Models/Task.php:94
      Illuminate\Foundation\Bootstrap\HandleExceptions::handleError("Attempt to read property "parentProject" on string", "[PATH]/things3-trello-sync-master/app/Models/Task.php")
  2   app/Models/Task.php:80
      App\Models\Task::mapToTrello("board")`

**Comment by stancl** [comment]
Hi, does either `things:tasks` or `things:list` work for you?

**Reply by sdquinn** [reply]
Not getting any output for things:list, but am for things:tasks. Should I provide that output here?

**Reply by sdquinn** [reply]
Here's things:tasks' output. [EDITED]

**Reply by sdquinn** [reply]
Also -- my apologies, things:list works! I just needed to provide the correct options for it, and it does in fact work. Here's things:list --areas for instance: [EDITED]

**Reply by stancl** [reply]
Had a quick look around the code and honestly not sure. I don't really use Things3 much anymore so it's possible their database schema changed in some way. If you confirm whether it's just recurring tasks I may be able to take a look at this later.

**Reply by sdquinn** [reply]
No worries. I think I'm seeing other recurring tasks -- at least ones that repeat on completion vs repeat on specific date -- come through, so think it's something else. Thanks for your help!
